server/src/main/java/com/objectcomputing/checkins/services/feedback_template/FeedbackTemplate.java

```diff
@@ -43,17 +46,19 @@
     private String description;
 
     @Column(name = "createdBy")
```

Comment:
Needs createdBy changed into lowercase createdby to avoid postgres issues.

Reply:
+1